theme/assets/scss/account.scss

```diff
@@ -240,14 +240,16 @@ tbody tr td:first-child {
     word-break: break-all;
 }
   
-/* for custom scrollbar for webkit browser*/  
+// for custom scrollbar for webkit browser
 ::-webkit-scrollbar {
     width: 1px;
 } 
 ::-webkit-scrollbar-track {
     -webkit-box-shadow: inset 0 0 1px rgba(0,0,0,0.3); 
+    box-shadow: inset;
```

Comment:
what it should do? 
-webkit-box-shadow == box-shadow
I think need only box-shadow

Reply:
> what it should do?
> -webkit-box-shadow == box-shadow
> I think need only box-shadow

Well it gives compatibility errors if both of them is not there so, feel free to do anything to fix that if you wish...